packages/client/screens/MusicPlaylistEditorCreationForm/MusicPlaylistEditorCreationFormName/index.tsx

```diff
@@ -0,0 +1,133 @@
+import { useSelector } from '@xstate/react';
+import { Text, View } from 'dripsy';
+import React from 'react';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { AppScreen, TextField } from '../../../components/kit';
+import MtvRoomCreationFormScreen from '../../../components/MtvRoomCreationForm/MtvRoomCreationFormScreen';
+import { useMpeRoomCreationFormActor } from '../../../hooks/useMusicPlaylistsActor';
+import { CreationMpeRoomFormActorRef } from '../../../machines/creationMpeRoomForm';
+
+export interface MusicTrackVoteCreationFormNameFormFieldValues {
+    roomName: string;
+}
+
+interface MusicTrackVoteCreationFormNameContentProps {
+    defaultRoomName: string;
+    handleGoBack: () => void;
+    handleGoNext: SubmitHandler<MusicTrackVoteCreationFormNameFormFieldValues>;
+}
+
+const MusicPlaylistEditorCreationFormNameContent: React.FC<MusicTrackVoteCreationFormNameContentProps> =
+    ({ defaultRoomName, handleGoBack, handleGoNext }) => {
+        const {
+            control,
+            handleSubmit,
+            formState: { errors },
+        } = useForm<MusicTrackVoteCreationFormNameFormFieldValues>();
+
+        return (
+            <MtvRoomCreationFormScreen
+                title="What is the name of the room?"
```

Comment:
I think for both Mtv and Mpe room creation form we should notify to the user in what type of room form he's in at least for the first screen ?